crypto/BRCryptoTransfer.c

```diff
@@ -117,7 +117,7 @@ extern BRCryptoTransfer
 cryptoTransferCreateAsBTC (BRCryptoUnit unit,
                            BRCryptoUnit unitForFee,
                            BRWallet *wid,
-                           BRTransaction *tid) {
+                           OwnershipKept BRTransaction *tid) {
     BRAddressParams  addressParams = BRWalletGetAddressParams (wid);
     BRCryptoTransfer transfer      = cryptoTransferCreateInternal (BLOCK_CHAIN_TYPE_BTC, unit, unitForFee);
     transfer->u.btc.tid = tid;
```

Comment:
So this assignment into `transfer` is holding a BRTransaction that is owned (`OwnershipKept`) by the caller.  The subsequent release doesn't call BRTransactionFee() - all consistent provided `tid` doesn't disappear...

Reply:
Correct, this functions like ETH now.